app/src/main/java/com/pokemonshowdown/app/BattleFragment.java

```diff
@@ -1755,7 +1755,8 @@ private AlertDialog parseMoveTargetDialog(final JSONArray active, final int move
         final int numFoes = foeIndex;
         final int currentActive = mCurrentActivePokemon;
         switch (target) {
-            case "normal":
+            case "any": //can hit anything on the BG
```

Comment:
Are you implying that `any` and `normal` are handled the same way? Doesn't sound like that to me.